lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -977,6 +977,33 @@ def test_get_core_file_api(self):
         self.assertEqual(process.GetCoreFile().GetFilename(), core_file_name)
         self.dbg.DeleteTarget(target)
 
+    @skipIfLLVMTargetMissing("X86")
+    def test_ro_cstring(self):
```

Comment:
Please expand "ro" to "read_only" in all the places you've used it, if you're referring to the flags of the memory mapping then fine then it's fine to use what the flag is, but otherwise, expanding the name saves future authors a few seconds figuring out what you mean.

Also if I'm going to grep for tests related to this topic, "ro" is not my first choice of search terms, though that is just me of course.